lib/matplotlib/tests/test_ticker.py

```diff
@@ -242,9 +245,8 @@ def test_set_params(self):
 class _LogitHelper:
     @staticmethod
     def isclose(x, y):
-        if x >= 1 or x <= 0 or y >= 1 or y <= 0:
-            return False
-        return np.isclose(-np.log(1/x-1), -np.log(1/y-1))
+        return (np.isclose(-np.log(1/x-1), -np.log(1/y-1))
+                if 0 < x < 1 and 0 < y < 1 else False)
```

Comment:
Not convinced that the ternary if-else expression makes this more readable. But no strong feeling.

Reply:
It's not so much about the ternary, more about '0 < x < 1' vs 'x >= 1 or x <= 0', I can revert to non-ternary if you prefer.

Reply:
Fully agree on `0 < x < 1`. Would slightly prefer non-ternary-if, but as said: no strong feelings.

Reply:
let's see what the next reviewer says, then

Reply:
It is fine this way. It is one less line of code, and one less `return` call.